packages/protocol/contracts/L1/IProofVerifier.sol

```diff
@@ -7,18 +7,17 @@
 pragma solidity ^0.8.20;
 
 /**
- * @title IProofVerifier Interface
- * @dev Interface for the ProofVerifier contract which is responsible for
- * verifying proofs.
+ * Contract that is responsible for verifying proofs.
+ * @title IProofVerifier
  */
 interface IProofVerifier {
     /**
-     * @notice Verify proof(s) for the given block.
```

Comment:
@d1onys1us so we are not using @notice anywhere? If you look at https://docs.soliditylang.org/en/v0.8.17/natspec-format.html, I think @notice is used many places.